frontend/src/pages/list/GenreList.tsx

```diff
@@ -105,6 +106,12 @@ export default function GenreList() {
   // Call to actually delete the element
   const deleteGenreFinal = () => {
     logger.debug("Delete Genre Finalized", selectedDeleteGenre);
+    GENRES_API.deleteGenre(selectedDeleteGenre.id);
+    const _genres = genres.filter(
+      (selectGenre) => selectedDeleteGenre.id != selectGenre.id
+    );
+    setGenres(_genres);
+    logger.debug("Delete Genre Completed", selectedDeleteGenre);
```

Comment:
We already have a "Delete Genre Finalized" log just above the code you added, and then this one here. Since we log the API call/response, I don't feel like we need both of these.